operator/v1/types_console.go

```diff
@@ -132,6 +132,10 @@ type ConsoleCustomization struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	AddPage AddPage `json:"addPage,omitempty"`
+	// perspectives allows enabling/disabling of perspective(s).
```

Comment:
Is there a way we can describe a perspective so that a user knows exactly what that means? How are they described in the user docs?